api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -124,8 +126,25 @@ message DownstreamTlsContext {
   OcspStaplePolicy ocsp_staple_policy = 8 [(validate.rules).enum = {defined_only: true}];
 }
 
+// TLS key log configuration.
+// The key log file format is "format used by NSS for its SSLKEYLOGFILE debugging output" (text taken from openssl man page)
+message TlsKeyLog {
+  // The path to save the TLS key log.
+  string tls_keylog_path = 1;
```

Comment:
Please add a validation option indicating that this field is required.

Reply:
I will add.

Reply:
@markdroth ,  it seems that there is no such  required validation for  a string in proto3. 
required validation can only be set for a message. Is it reasonable to use a new message for tls_keylog_path ?

Reply:
Try using `[(validate.rules).string = {min_len: 1}]`.